recipes/toolchain/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: toolchain
```

Comment:
I'd prefer not to. It will already be in the `conda-forge` channel.

Besides, if people are wanting to force their own `toolchain` package to be installed instead of this one there are other mechanisms ( https://github.com/conda/conda/pull/2323 ) that are being put in place for that exact purpose.